### ISSUE TITLE
License information is unclear or unknown for many standards documents

### ISSUE THREAD

**Issue by baskaufs** [tdwg/rs.tdwg.org#6]
In many of the prior standards, no information is given about document licensing.  In some cases, copyright is asserted, in others it is not.  In some cases, (I think Audubon Core) there is a default license asserted by the displaying platform that may not conform to the actual practice of TDWG.  This issue is complicated by the fact that TDWG does not actually own the copyright for many of the prior standards.  In some cases, they were written independently of TDWG and adopted as standards after the fact and in other cases, they were written on behalf of TDWG, but copyright was asserted by the publisher (who was not TDWG).  To make matters worse, at least one is behind an expensive paywall and in other cases print documents are out of print.  TDWG should consider retiring some of these standards if free access to them by the public cannot be assured.  @peterdesmet

**Comment by ramonawalls** [comment]
It is also important to add a license to this repo, so that people know how they can reuse the contents. It sounds like it might be necessary to add different licenses to different name spaces. As it currently stands (i.e. unlicensed), the RDF is technically unusable for open science.

**Comment by baskaufs** [comment]
Good point, @ramonawalls. I'll try to find out if there is any existing policy on this. I believe that TDWG documents are supposed to be CC BY, but for stuff like this that is purely data, CC0 seems like the appropriate license to me. One issue is that there are a number of scripts in here for convenience. I suppose they could have licenses that are not the repo default if necessary.

It would be good to have some place where there is a record of TDWG-wide policies like this. If there is such a place, I don't know of it. Maybe something that the @tdwg/tag-ig should manage

**Comment by ramonawalls** [comment]
For the record, I support having the standards and especially the RDF format available as CC0. We should encourage people to reuse the TDWG IRIs as the best way to cite and give credit.

**Comment by baskaufs** [comment]
One of the requirements of the Standards Documentation Specification [section 4](https://github.com/tdwg/vocab/blob/master/sds/documentation-specification.md#4-machine-readable-documents) is that "the set of machine-readable relationships (a graph in RDF) SHOULD be unchanged within a version of a document or vocabulary. Regardless of the serialization in which the machine-readable metadata are provided, that serialization MUST parse to the same set of relationships (i.e. the same RDF graph)." The purpose of this repository (rs.tdwg.org) is to make that possible by generating every serialization of the standards metadata from a single source data. Thus, I think that we should assume that applying a CC0 license to the source data here would imply that all RDF serializations generated from those data would also be under a CC0 license. But it would be good to have an explicit statement about that.
